README.md

```diff
@@ -169,10 +170,63 @@ ATTR{suspend_mode}=="[oem|keep|off]"
 ```
 
 As en example, to set `suspend_mode`to `off`:
-`ACTION=="add|change", KERNEL=="multicolor:chassis", SUBSYSTEM=="leds", ATTR{suspend_mode}="off"
+`ACTION=="add|change", KERNEL=="ayaneo:rgb:joystick_rings", SUBSYSTEM=="leds", ATTR{suspend_mode}="off"
 `
 
 to create the rule in one line:
 ```shell
-$ echo 'ACTION=="add|change", KERNEL=="multicolor:chassis", SUBSYSTEM=="leds", ATTR{suspend_mode}="off"' | sudo tee /etc/udev/rules.d/99-led_suspend_mode.rules
+$ echo 'ACTION=="add|change", KERNEL=="ayaneo:rgb:joystick_rings", SUBSYSTEM=="leds", ATTR{suspend_mode}="off"' | sudo tee /etc/udev/rules.d/99-led_suspend_mode.rules
 ```
+
+### Bypass Charge Control
+
+Bypass Charge can be enabled, disabled, start and stop battery level can be set via sysfs.
+
+n most systems this will be mounted at `/sys/class/power_supply/ayaneo:bypass_charge/` and provides the following files:
+
+#### charge_type
+
+Read/write.
+
+Enables or disables the bypass charge. Accepts a value of 6 and 0 or "Custom" and "Unknown".
```

Comment:
We should only permit the common names to be written, not the int. It's confusing to accept both, and 0/6 have no tangible meaning outside the firmware implementation.

We should also have a read only `charge_control_mode_choices` attribute that provides the available options.

Reply:
I just follow and use the kernel driver docs and they allowed both and I cannot change it, cause it is handled in the kernel.